CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+### citus v10.0.3 (March 16, 2021) ###
+
+* Prevents infinite recursion for queries that involve `UNION ALL`
+  below `JOIN`
+
+* Fixes a crash in queries with a modifying `CTE` and a `SELECT`
+  without `FROM`
+
+* Adds GUC to set maximum connection lifetime
```

Comment:
maybe:

"Adds `citus.max_cached_connection_lifetime` GUC to set maximum connection lifetime"

so that users can know which udf it is